nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/processors/excel/SplitExcel.java

```diff
@@ -66,10 +74,10 @@
 @Tags({"split", "text"})
 @InputRequirement(InputRequirement.Requirement.INPUT_REQUIRED)
 @CapabilityDescription("This processor splits a multi sheet Microsoft Excel spreadsheet into multiple Microsoft Excel spreadsheets where each sheet from the original" +
-        " file is converted to an individual spreadsheet in its own flow file. Currently this processor is only capable of processing .xlsx" +
-        " (XSSF 2007 OOXML file format) Excel documents and not older .xls (HSSF '97(-2007) file format) documents." +
-        " Please note all original cell styles are dropped and formulas are removed leaving only the calculated values." +
-        " Even a single sheet Microsoft Excel spreadsheet is converted to its own flow file with all the original cell styles dropped and formulas removed."
+        " file is converted to an individual spreadsheet in its own flow file. This processor is capable of processing both password and non password protected" +
+        " .xlsx (XSSF 2007 OOXML file format) and older .xls (HSSF '97(-2007) file format) Excel documents." +
```

Comment:
`HSSF` is the internal Apache POI name, which should not be used in the component documentation.

```suggestion
        "modern XLSX and older XLS Excel spreadsheets." +
```